tox.ini

```diff
@@ -130,9 +130,8 @@ commands_pre =
   exporter-otlp-proto-grpc: pip install {toxinidir}/exporter/opentelemetry-exporter-otlp-proto-common
   exporter-otlp-proto-grpc: pip install {toxinidir}/exporter/opentelemetry-exporter-otlp-proto-grpc
 
-  exporter-otlp-proto-http: pip install {toxinidir}/opentelemetry-proto
-  exporter-otlp-proto-http: pip install {toxinidir}/exporter/opentelemetry-exporter-otlp-proto-common
-  exporter-otlp-proto-http: pip install {toxinidir}/exporter/opentelemetry-exporter-otlp-proto-http[test]
+  proto3-opentelemetry-exporter-otlp-proto-http: pip install -r {toxinidir}/exporter/opentelemetry-exporter-otlp-proto-http/test-requirements-0.txt
+  proto4-opentelemetry-exporter-otlp-proto-http: pip install -r {toxinidir}/exporter/opentelemetry-exporter-otlp-proto-http/test-requirements-1.txt
```

Comment:
Question: if you change these in `commands_pre`, should you alse change the stuff with the same name in `commands`?

Reply:
No, because the matching stuff in `commands` is the same for `proto3*` and `proto4*`.